aws/data_source_aws_prefix_list_test.go

```diff
@@ -74,27 +76,31 @@ func testAccDataSourceAwsPrefixListCheck(name string) resource.TestCheckFunc {
 }
 
 const testAccDataSourceAwsPrefixListConfig = `
-data "aws_prefix_list" "s3_by_id" {
-  prefix_list_id = "pl-68a54001"
-}
+data "aws_region" "current" {}
 
 data "aws_prefix_list" "s3_by_name" {
-  name = "com.amazonaws.us-west-2.s3"
+  name = "com.amazonaws.${data.aws_region.current.name}.s3"
```

Comment:
Not for this PR, but might be good to add a reverse DNS prefix attribute to the `aws_partition` data source since it varies across partitions outside AWS Commercial and GovCloud (US).

Reply:
Sort of related, there's also the issue of EC2 compute hostnames/domains. In #16582, I created `resourceAwsEc2RegionalPrivateDnsSuffix()` and `resourceAwsEc2RegionalPublicDnsSuffix()`. That approach is more clumsy than adding fields like `ec2_private_compute_domain` and `ec2_public_compute_domain` to `aws_region`. Thoughts?

Reply:
See #16638 for the DNS prefix